packages/mobile/android/gradle.properties

```diff
@@ -34,3 +34,5 @@ android.bundle.enableUncompressedNativeLibs=false
 # You can also override it from the CLI using
 # ./gradlew <task> -PreactNativeArchitectures=x86_64
 reactNativeArchitectures=armeabi-v7a,arm64-v8a
+
+NDK_PATH=~/Android/Sdk/ndk/21.4.7075529
```

Comment:
Should it be commited?

Reply:
Good catch! I commited it accidentaly, gonna remove it asap